### PR TITLE
W01L03's Slides

### PR DESCRIPTION
**Pull request by arrowrowe**

### CODE REVIEW

**Comment by at15** [comment]
\w/ 前排围观

File: web/2016_Spring/js-start/slide.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                <h1 style="color:red">ＳＯＳ団のサイトにようこそ！</h1>
+                <p><img src="http://www.haruhi.tv/img/zozlogo.jpg" alt="SOS Logo" title="Hey"></p>
+                <h2><a href="http://www.haruhi.tv/" target="_blank">入り口</a></h2>
+                <p>
+                  メールはこちらから→
+                  <img src="http://www.haruhi.tv/img/post_no.jpg" alt="In Construction">
+                </p>
+              </div>
+            </body>
+            </html>
```
Comment (ComMouse) [inline web/2016_Spring/js-start/slide.html:119]: ....?

Comment (at15) [inline web/2016_Spring/js-start/slide.html:119]: ....

Comment (arrowrowe) [inline web/2016_Spring/js-start/slide.html:119]: @ComMouse just s sample html.........

**Comment by at15** [comment]
LGTM

**Pull request merged**